typescript-service-generator-core/src/main/java/com/palantir/code/ts/generator/ServiceClassParser.java

```diff
@@ -60,26 +61,35 @@
     }
 
     public ServiceModel parseServiceClass(Class<?> serviceClass, TypescriptServiceGeneratorConfiguration settings) {
+        return this.parseServiceClass(Lists.newArrayList(serviceClass), settings);
+    }
+
+    public ServiceModel parseServiceClass(List<Class<?>> serviceClasses, TypescriptServiceGeneratorConfiguration settings) {
```

Comment:
1. Can we do a serviceClass... method to allow me to pass in two classes without constructing a collection?
2. Does order matter here? (is there a reason for list over Collection or even Iterable?)
3. Not sure what convention we want for naming but it seems odd to pass multiple classes to something that is named parseServiceClass (singular)